plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -479,6 +479,7 @@ public static <T> Map<CanonicalColumnName, T> toCanonicalNameKeyedMap(Map<String
     public static <T> Map<CanonicalColumnName, T> toCanonicalNameKeyedMap(Map<String, T> map, Map<String, CanonicalColumnName> canonicalColumnNames)
     {
         return map.entrySet().stream()
+                .filter(entry -> entry.getValue() != null)
```

Comment:
Can we remove another `toCanonicalNameKeyedMap(Map<String, T> map)` in a preparatory commit?